Resources/Private/Templates/Search/Results.html

```diff
@@ -37,8 +37,16 @@
 		</f:if>
 
 		<f:if condition="{resultSet.usedSearch.numberOfResults}">
+			
 			<span class="result-found">
-				<s:translate key="results_found" arguments="{0:resultSet.usedSearch.numberOfResults, 1: resultSet.usedSearch.queryTime}">Found %d results in %d seconds</s:translate>
+				<f:if condition="{resultSet.usedSearch.numberOfResults -> f:count()} == 1">
```

Comment:
usedSearch->getNumberOfResults() should allready return the count, why is this required? The test also shows an error that the argument of the count ViewHelper should not be an integer. I think this should be enought:

```php
<f:if condition="{resultSet.usedSearch.numberOfResults} == 1">
```

Reply:
Ah ok, I think I oversaw this. Can you push to my branch and make the needed change? Otherwise I'll do the change this evening.